### PR TITLE
feat: Optimize color of `Submit to contest` button

### LINKED ISSUE #386: Optimize color of `Submit to contest` button
The colorful `Submit to content` button may be more effective if it matches all the other buttons in the flow. We could change the button to be the gradient red from the tutorial flow. The effect of this would be a reduction in the dropoff from the editor to contest submission, where we lose 80% of users.

### PR DESCRIPTION
**Pull request by abarmat**
Closes #386

### CODE REVIEW

**Comment by belohlavek** [comment]
@abarmat Make sure to remove the unused image (if truly unused)

File: src/components/ContestButton/ContestButton.css [hunk 1]
```diff
@@ -9,8 +9,7 @@
 .ContestButton.ui.button.mini:active,
 .ContestButton.ui.button.mini:focus,
 .ContestButton.ui.button.mini:hover {
-  background-image: url('../../images/contest-button.png');
-  background-size: 100%;
+  background-color: #ff2d55;
```
Comment (belohlavek) [inline src/components/ContestButton/ContestButton.css]: This color should be taken from the CSS variable

**Comment by belohlavek** [comment]
Are we naming it Add to Contest again?

File: src/components/ContestButton/ContestButton.css (hunk 1 above)
Comment (abarmat) [reply src/components/ContestButton/ContestButton.css]: Will do

**Pull request merged**